src/validation/index.ts

```diff
@@ -26,13 +26,6 @@ export const validatorMiddleware: AdapterMiddlewareBuilder =
       })
     }
 
-    if (!req.body) {
-      throw new AdapterInputError({
-        message: 'Body not present in adapter request, returning 400',
-        statusCode: 400,
-      })
-    }
-
```

Comment:
Note: We already have a test for application/json header. If the header is present with no body, fastify errors so it will never reach this part.